core/src/org/pentaho/di/core/Const.java

```diff
@@ -1023,6 +1023,12 @@ public String getMessage() {
   public static final String KETTLE_SPLIT_FIELDS_REMOVE_ENCLOSURE = "KETTLE_SPLIT_FIELDS_REMOVE_ENCLOSURE";
 
   /**
+   * The name of the variable that switches off fix for calculation of timezone decomposition.
+   */
+  public static final String KETTLE_OLD_DATE_CALCULATION_TIMEZONE_DECOMPOSITION =
```

Comment:
Can we update the variable name and value to use the existing naming standard for backwards-compatibility switches?

`KETTLE_COMPATIBILITY_*`

https://github.com/mchen-len-son/pentaho-kettle/blob/a06ded8a52b1bde7c56e04d4cc7bf91eca157f09/core/src/org/pentaho/di/core/Const.java#L731